src/mongocrypt-marking.c

```diff
@@ -512,120 +550,134 @@ _mongocrypt_fle2_placeholder_common (_mongocrypt_key_broker_t *kb,
 }
 
 
+// Shared implementation for insert/update and insert/update ForRange
 static bool
-_mongocrypt_fle2_placeholder_to_insert_update_ciphertext (
+_mongocrypt_fle2_placeholder_to_insert_update_common (
    _mongocrypt_key_broker_t *kb,
-   _mongocrypt_marking_t *marking,
-   _mongocrypt_ciphertext_t *ciphertext,
+   mc_FLE2InsertUpdatePayload_t *out,
+   int64_t *contentionFactor,
```

Comment:
```suggestion
```

Callers do not use the resulting chosen `contentionFactor`.

Suggest also renaming the parameter `maxContentionCounter => contentionFactor` in `_mongocrypt_fle2_placeholder_common`.